### PR TITLE
[Term entry] Python:Numpy Math Function: .hypot()

### PR DESCRIPTION
**Pull request by JMhuautla**
### Description

- Added the hypot() term to Python:Numpy math-methods
- Closes issue #5235 

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by mamtawardhani** [comment]
Hey @JMhuautla looks like the entry is empty, can you please check it on your local device and push the content?

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+Title: '.hypot()'
+Description: 'Returns the hypotenuse, given the "legs" of a triangle or triangles.'
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:3]:
```suggestion
Description: 'Returns the hypotenuse of a right triangle, given the "legs".'

```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  - 'Math'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+In [Numpy](https://www.codecademy.com/resources/docs/numpy), given the "legs" of a right triangle, the **`.hypot`** function returns the hypotenuse.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:18]:
```suggestion
In [Numpy](https://www.codecademy.com/resources/docs/numpy), given the "legs" of a right triangle, the **`.hypot()`** function returns the hypotenuse.
```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+## Syntax
+
+```pseudo
+numpy.hypot(x1, x2, out=None, where=True)
+```
+
+- `x1`, `x2`: Leg of the triangle(s), respectively. If both shapes are not indentical, they must be broadcastable to a common shape.
+- `out`: An optional parameter that allows us to store the output array if the location is specified. If provided, it must have a shape that the inputs broadcast to. If not provided or None, a new array will be allocated for the result.
+- `where`: An array-like optional parameter that determines the elements on which the method is to be applied. At locations where the condition is True, the _out_ array will be set to the ufunc result.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:28]:
```suggestion
- `x1`, `x2`: These are the "legs" of the triangle(s). If the shapes of `x1` and `x2` are not identical, they must be broadcastable to a common shape.
- `out` (Optional): This parameter specifies an array where the result will be stored. The shape of `out` must match the shape of the output. If not provided, a new array will be allocated for the result.
- `where` (Optional): An optional condition that can be applied element-wise on the input arrays. The result will only be computed where the condition is `True`. If the condition is `False` for an element, that element’s result will be ignored.
```

Comment (mamtawardhani) [reply content/numpy/concepts/math-methods/terms/hypot/hypot.md:28]: reframed for better readability

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+ ## Example 1
+
+The below example calculates the hypotenuse of one right triangle.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:32]:
```suggestion
The below example calculates the hypotenuse of one right triangle:
```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+```py
+# Importing the numpy library
+import numpy as np
+
+print("If the first leg is 3 and the second is 4, the hypotenuse is ",np.hypot(3, 4))
+```
+
+The code above generates  the following output:
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:41]:
```suggestion
The code above generates the following output:

```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+```shell
+If the first leg is 3 and the second is 4, the hypotenuse is 5
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:44]:
```suggestion
If the first leg is 3 and the second is 4, the hypotenuse is  5.0
```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+## Example 2
+
+The below example calculates the hypotenuses of 3 right triangles
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:49]:
```suggestion
The following example calculates the hypotenuses of three right triangles:
```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+```shell
+Output array: [ 5. 13. 10.]
+
+The hypotenuses are 5.0, 13.0 and 10.0, respectively.
+```
+
+## Codebyte Example
+
+In this codebyte, `hypot()` computes the hypotenuses for two triangles. The first one has legs of 2 and 3, and the second one legs of 8 and 15.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:76]:
```suggestion
In this codebyte, `.hypot()` computes the hypotenuses for two triangles. The first triangle has legs of *2* and *3*, and the second triangle has legs of *8* and *15*.
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @JMhuautla, thank you for contributing to Codecademy Docs, the entry is nicely written! 😄 

I've suggested a few changes, could you please review and modify those at your earliest convenience? Thank you! 😃

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the changes @JMhuautla 😄 

The entry looks good for a second round of review 🚀

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
+---
+Title: '.hypot()'
+Description: 'Returns the hypotenuse of a right triangle, given the "legs".'
```
Comment (PragatiVerma18) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:3]: According to the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md), we should not use apostrophes or quotes in Title: or Description:.

```suggestion
Description: 'Returns the hypotenuse of a right triangle, given the legs.'
```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+# Creating the sides of three triangles
+array1 = [3,5,6]
+array2 = [4,12,8]
+
+# Computing the hypotenuses
+hypotenuses = np.hypot(array1, array2)
+
+print("Output array:", hypotenuses)
+print("\nThe hypotenuses are {}, {} and {}, respectively.".format(hypotenuses[0], hypotenuses[1], hypotenuses[2]))
+```
```
Comment (PragatiVerma18) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:66]: We should use more relatable comments and variable names to make the code example easier to understand -

```suggestion
import numpy as np

# Creating the sides of three triangles
triangle_leg1 = [3,5,6]  # First leg of each triangle
triangle_leg2 = [4,12,8]  # Second leg of each triangle

# Computing the hypotenuses
hypotenuses = np.hypot(array1, array2)

print("Calculated hypotenuses:", hypotenuses)
print("\nThe hypotenuses are {}, {} and {}, respectively.".format(hypotenuses[0], hypotenuses[1], hypotenuses[2]))
```

```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+
+The code above generates the following output:
+
+```shell
+Output array: [ 5. 13. 10.]
+
+The hypotenuses are 5.0, 13.0 and 10.0, respectively.
```
Comment (PragatiVerma18) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:73]: Based on above code changes, output will also change as following - 

```suggestion
Calculated hypotenuses: [ 5. 13. 10.]

The hypotenuses are 5.0, 13.0 and 10.0, respectively.

```

File: content/numpy/concepts/math-methods/terms/hypot/hypot.md [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+```codebyte/python
+import numpy as np
+
+# Input Array
+array1 = [2, 8]
+array2 = [3, 15]
+
+result = np.hypot(array1, array2)
+
+print(result)
```
Comment (PragatiVerma18) [inline content/numpy/concepts/math-methods/terms/hypot/hypot.md:89]: Similar suggestions as above example - we should use more descriptive/relatable variable names and comments for more clarity.

```suggestion
import numpy as np

first_leg = [2, 8]  # First leg of each triangle
second_leg = [3, 15]  # Second leg of each triangle

# Calculate the hypotenuses for each triangle
hypotenuses = np.hypot(first_leg, second_leg)

print("The calculated hypotenuses are:", hypotenuses)
```

**Review by PragatiVerma18 — CHANGES_REQUESTED** [review]
Hey @JMhuautla, the entry looks good, I have some suggestions, please go through them and make the required changes. Do let me know if you have any questions. Thanks.

**Review by PragatiVerma18 — APPROVED** [review]

**Pull request merged**